samples/word/99-preview-apis/manage-comments.yaml

```diff
@@ -0,0 +1,235 @@
+id: word-manage-comments
+name: Manage comments
+description: Basic operations using comments.
+host: WORD
+api_set:
+    WordApi: '1.4'
+script:
+    content: |
+        $("#insert").click(() => tryCatch(insertComment));
+        $("#reply").click(() => tryCatch(replyToFirstCommentInSelection));
+        $("#get-comments").click(() => tryCatch(getComments));
+        $("#get-comments-in-selection").click(() => tryCatch(getCommentsInSelection));
+        $("#edit").click(() => tryCatch(editFirstCommentInSelection));
+        $("#resolve").click(() => tryCatch(resolveFirstCommentInSelection));
+        $("#delete").click(() => tryCatch(deleteFirstCommentInSelection));
+        $("#range").click(() => tryCatch(getFirstCommentRange));
+        $("#setup").click(() => tryCatch(setup));
+
+        async function insertComment() {
+          await Word.run(async (context) => {
+            let text = $("#comment-text")
+              .val()
+              .toString();
+            let comment = context.document.getSelection().insertComment(text);
+            comment.load("*");
+            await context.sync();
+            console.log("Comment inserted:");
+            console.log(comment);
```

Comment:
Nit: Let's make this one `log` statement.

Reply:
Ended up keeping the extra log statement because I preferred how the log is formatted in the Script Lab and its interactivity